.gitlab-ci.yml

```
@@ -0,0 +1,16 @@
+image: node:14.5.3
+stages:
+  - test
+
+variables:
+  PROJECT_NAME: algorithmia
+  DOCKER_HOST: tcp://docker:2375/
+  DOCKER_DRIVER: overlay2
+  RUNNING_ON_BUILD_SERVER: "true"
+
+test:
+  stage: test
+  script:
+  - npm install mocha-headless-chrome -g 
+  - if [ -z $ALGORITHMIA_DEFAULT_API_KEY ]; then echo "Algorithmia API key not defined"; exit 1; fi
+  - npm test --key=$ALGORITHMIA_DEFAULT_API_KEY
```

As long as GitLab has an `API_KEY` env var defined:

```suggestion
  - npm test
```